packages/combobox/src/Combobox/Combobox.tsx

```diff
@@ -1150,77 +1136,86 @@ export function Combobox<M extends boolean>({
   } as const;
 
   return (
-    <LeafyGreenProvider darkMode={darkMode}>
-      <ComboboxContext.Provider
-        value={{
-          multiselect,
-          size,
-          withIcons,
-          disabled,
-          isOpen,
-          state,
-          searchState,
-          chipTruncationLocation,
-          chipCharacterLimit,
-          inputValue,
-          overflow,
-          popoverZIndex,
-        }}
+    <ComboboxContext.Provider
+      value={{
+        multiselect,
+        size,
+        withIcons,
+        disabled,
+        isOpen,
+        state,
+        searchState,
+        chipTruncationLocation,
+        chipCharacterLimit,
+        inputValue,
+        overflow,
+        popoverZIndex,
+      }}
+    >
+      <FormField
+        label={label}
+        description={description}
+        errorMessage={errorMessage}
+        state={state}
+        size={size}
```

Comment:
Left a Chromatic review. XS variant looks like it needs extra padding for parity

Reply:
Fixed this take a look?